packages/common/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@sphereon/openid4vci-common",
+  "version": "0.0.1",
+  "description": "",
+  "main": "index.ts",
```

Comment:
This is incorrect. Never use a TS file as a main entry point. Look at the other package.jsons